cmd/run/run.go

```diff
@@ -410,6 +413,20 @@ func run(_ *cobra.Command, _ []string) {
 	}
 }
 
+func readCertificate(ctx context.Context, certPath, keyPath string) (*certwatcher.CertWatcher, error) {
+	watcher, err := certwatcher.New(certPath, keyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	go func() {
+		if err := watcher.Start(ctx); err != nil {
+		}
```

Comment:
Do we at least want to log the error if not log Fatal?

Reply:
Done. Added logger.Fatal here